packages/fx-core/src/common/templates-config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "^0.9.0",
+    "version": "0.0.0-rc",
```

Comment:
is it correct?

Reply:
right

Reply:
templates only contains 0.0.0-rc 0.0.0-alpha as prerelease version